code/modules/jobs/jobs.dm

```diff
@@ -114,9 +137,15 @@ GLOBAL_LIST_INIT(security_positions, list(
 	JOB_NAME_DEPUTY
 ))
 
-GLOBAL_LIST_INIT(security_lightup_areas, typecacheof(list(
-	/area/security
-)))
+GLOBAL_LIST_INIT(security_lightup_areas,	\
+	typecacheof(list(						\
+		/area/security						\
+	)) - typecacheof(list(					\
```

Comment:
I don't think this works, you can just subtract the non-typecache version and it should remove the keys

Reply:
It does work. I've seen this done in other places.

[Proof](https://discord.com/channels/427337870844362753/943313997334208542/1134669663217930340): 
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/65794972/658e31fd-622a-4f42-a524-626f776c1d71)